src/linter.js

```diff
@@ -39,8 +38,6 @@ export default function linter(key, options, compilation) {
   /** @type {Promise<LintResult[]>[]} */
   const rawResults = [];
 
-  const crossRunResultStorage = getResultStorage(compilation);
```

Comment:
This cache held all results that had been computed... invalidations were supposed to be deleted from this and re-run.. so we don't recompute results for files that did not change... but still show them if they had issues.

Reply:
Related #83